landing-page/index.html

```diff
@@ -0,0 +1,33 @@
+<html>
+  <head>
+    <style>
+      body {
+  background-color: MistyRose;
+}
+      </style>
+  </head>
+  <body>
+      <style>
+          h1{
+  background-color: white;
+  color: MistyRose;
+  font: 60pt Impact, Charcoal, sans-serif;
+  display: flex;
+  justify-content: center;
+}
+h2{
+  display: flex;
+  justify-content: center;
+}
+p{
+  margin-top: 70px;
+  margin-left: 60px;
+  margin-right: 450px;
```

Comment:
It's really tempting to position things this way, but flexbox is a much better way! Have you seen https://flexboxfroggy.com/? :)